packages/service-library/tests/rabbitmq/test_rabbitmq.py

```diff
@@ -462,3 +462,45 @@ async def test_unsubscribe_consumer(
     await client.unsubscribe(exchange_name)
     with pytest.raises(aio_pika.exceptions.ChannelNotFoundEntity):
         await client.unsubscribe(exchange_name)
+
+
+@pytest.mark.parametrize("max_requeue_retry", [None, 3, 10])
+@pytest.mark.no_cleanup_check_rabbitmq_server_has_no_errors()
```

Comment:
why do you put that mark here? do we have unhandled errors in rabbit when you run that test? what kind of?

Reply:
just copied it, removed it since it made no sense to have it here